### ISSUE TITLE
Connect to existing wgeasy

### ISSUE THREAD

**Issue by sahara101** [donaldzou/WGDashboard#308]
Hello! This looks really nice :)

Is there a way to connect to an existing weejewel/wg-easy installation? Or is the server list only for WGDashboard servers. 

Thanks!

**Comment by donaldzou** [comment]
Hi! I'm not sure how wg-easy configure WG configuration, if they're under the default WireGuard configuration location, and you can view them by typing `wg show conf_name`, then yes WGDashboard can find and manage them :)

**Comment by sahara101** [comment]
I just connected to the conainer

`bash-5.0# wg show --help
Usage: wg show { <interface> | all | interfaces } [public-key | private-key | listen-port | fwmark | peers | preshared-keys | endpoints | allowed-ips | latest-handshakes | transfer | persistent-keepalive | dump]`

`bash-5.0# cd /etc/wireguard/
bash-5.0# ls
wg0.conf                   wg0.json                   wireguard-module-synology`

But I do not know what to write in the API field :D

**Comment by donaldzou** [comment]
Hi, what do you mean by API field?

**Comment by sahara101** [comment]
Hi,

Docu mentions:

Starting with v4.0, you can access WGDashboards on other server through one WGDashboard with API Keys

**Comment by donaldzou** [comment]
Yes, but you don't have to. You can login to the dashboard with `admin` and `admin` as username and password by default.

**Comment by sahara101** [comment]
I meant if we are able to connect wg-easy to wgdashboard. No prob if that does not work/will not be implemented

**Comment by DaanSelen** [comment]
Hello @sahara101 please clarify what you mean with "connect", do you mean:

1. Configure WGDashboard from WGeasy or the other way around?
2. Configure WGDashboard AND WGeasy seperately from a remote server?

Please add some context, examples like what you want to do can also help! We always like to help.

**Comment by sahara101** [comment]
Well since I saw you have the option to connect to a remote WGDashboard, I thought maybe it would also work to connect to a remote wg-easy container. So I could use WGDashboard as the frontpage of wg-easy as well, too see the wg-easy connection/peers.  Hope this clarifies what I mean.

**Comment by DaanSelen** [comment]
Hey yes, I kind of see what you mean, but wg-easy if not really a different product than WGDashboard, they are both applications to manage the underlying WireGuard application. They serve the same purpose. They are direct alternatives.

If you mean you want to let WGDashboard be able to manage an already existing WIREGUARD (not wgeasy) environment, then yes this could be possible if the configurations are present on the filesystem!

Feel free to ask more questions if you'd like.

**Comment by sahara101** [comment]
They are present on a remote system, also through docker. I think this complicates things.

**Comment by DaanSelen** [comment]
If you can export the configurations from WGEasy and then you can get them working on WGDashboard if you'd like!

**Comment by donaldzou** [comment]
Hi! I will be closing this as wg-easy and WGDashboard are two different projects and how it designs.
